packages/kit/test/apps/basics/test/test.js

```diff
@@ -864,7 +864,7 @@ test.describe('Actions', () => {
 			const log_promise = page.waitForEvent('console');
 			const button = page.locator(`#${id}`);
 			await button.click();
-			expect(await button.textContent()).toBe('processed'); // needed to make sure action completes
+			await page.waitForSelector(`button#${id}[data-processed="true"]`);
```

Comment:
Can you reuse the locator here?

I don't remember the exact method, but there should be something like toHaveAttribute
```suggestion
			await expect(button).toHaveAttribute('data-processed', 'true')
```

Reply:
It was `toHaveAttribute` indeed.